vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/create/ProjectInformationDialog.java

```diff
@@ -150,7 +150,8 @@ public void loadOfferContent(Offer offer) {
       titleField.setValue(offer.projectTitle().title());
       projectObjective.setValue(offer.projectObjective().objective());
       offer.experimentalDesignDescription()
-          .ifPresent(it -> experimentalDesignField.setValue(it.description()));
+          .ifPresentOrElse(it -> experimentalDesignField.setValue(it.description()),
+              () -> experimentalDesignField.setValue(""));
```

Comment:
```suggestion
          .ifPresentOrElse(it -> experimentalDesignField.setValue(it.description()),
              experimentalDesignField::clear);
```
You can use Vaadin's clear method to ensure that placeholders or empty values are shown as configured for the field.